multiproof.go

```diff
@@ -54,7 +72,9 @@ func CreateMultiProof(transcript *common.Transcript, ipaConf *ipa.IPAConfig, Cs
 		return nil, errors.New("cannot create a multiproof with 0 queries")
 	}
 
-	banderwagon.BatchNormalize(Cs)
+	if err := banderwagon.BatchNormalize(Cs); err != nil {
+		return nil, fmt.Errorf("could not batch normalize commitments: %w", err)
+	}
```

Comment:
Wasn't a big deal to add the `error` (for points at infinity) since `CreateMultiProof` already returned an error.

Despite the client would still have to handle this situation (which must not happen), at least their process won't panic and can close gracefully.